packages/loot-design/src/components/sidebar.js

```diff
@@ -541,6 +537,12 @@ function Tools() {
             to="/rules"
             indent={15}
           />
+          <SecondaryItem
```

Comment:
Could we have it below "repair split transactions"? IMO it should be last item in this menu

Reply:
It will be after #456 :)